src/server/game/Handlers/MiscHandler.cpp

```diff
@@ -543,13 +543,20 @@ void WorldSession::HandleSetSelectionOpcode(WorldPacket & recv_data)
 
 void WorldSession::HandleStandStateChangeOpcode(WorldPacket & recv_data)
 {
-#if defined(ENABLE_EXTRAS) && defined(ENABLE_EXTRA_LOGS)
-    // too many spam in log at lags/debug stop
-    sLog->outDebug(LOG_FILTER_NETWORKIO, "WORLD: Received CMSG_STANDSTATECHANGE");
-#endif
     uint32 animstate;
     recv_data >> animstate;
 
+    switch (animstate)
+    {
+        case UNIT_STAND_STATE_STAND:
+        case UNIT_STAND_STATE_SIT:
+        case UNIT_STAND_STATE_SLEEP:
+        case UNIT_STAND_STATE_KNEEL:
+            break;
+        default:
+            return;
+    }
+
     _player->SetStandState(animstate);
```

Comment:
it would look more logical to move `_player->SetStandState(animstate);` inside the `switch`

Reply:
No its must be after switch

Reply:
ok for me but, just for my understanding, is it only about codestyle or I'm missing something?

Reply:
set the state must handle after we get list of allowed stand states